tests/github-action-helper.sh

```diff
@@ -2,31 +2,65 @@
 
 set -eEuo pipefail
 
+: "${FUNCTION:=${1}}"
+
+# source https://krew.sigs.k8s.io/docs/user-guide/setup/install/
+install_krew() {
+  # cd "$(mktemp -d)"
+  OS="$(uname | tr '[:upper:]' '[:lower:]')"
+  ARCH="$(uname -m | sed -e 's/x86_64/amd64/' -e 's/\(arm\)\(64\)\?.*/\1\2/' -e 's/aarch64$/arm64/')"
+  curl -fsSLO "https://github.com/kubernetes-sigs/krew/releases/latest/download/krew.tar.gz"
+  tar zxvf krew.tar.gz
+  KREW=./krew-"${OS}_${ARCH}"
+  "$KREW" install krew
+}
+
 # source https://github.com/rook/rook
 use_local_disk() {
-    sudo swapoff --all --verbose
-    sudo umount /mnt
-    # search for the device since it keeps changing between sda and sdb
-    sudo wipefs --all --force /dev/$(lsblk|awk '/14G/ {print $1}'| head -1)1
-    sudo lsblk
+  sudo swapoff --all --verbose
+  sudo umount /mnt
+  # search for the device since it keeps changing between sda and sdb
+  sudo wipefs --all --force /dev/"$(lsblk|awk '/14G/ {print $1}'| head -1)"1
+  sudo lsblk
 }
 
 deploy_rook() {
-    kubectl create -f https://raw.githubusercontent.com/rook/rook/master/cluster/examples/kubernetes/ceph/common.yaml
-    kubectl create -f https://raw.githubusercontent.com/rook/rook/master/cluster/examples/kubernetes/ceph/crds.yaml
-    kubectl create -f https://raw.githubusercontent.com/rook/rook/master/cluster/examples/kubernetes/ceph/operator.yaml
-    curl https://raw.githubusercontent.com/rook/rook/master/cluster/examples/kubernetes/ceph/cluster-test.yaml -o cluster-test.yaml
-    sed -i "s|#deviceFilter:|deviceFilter: $(lsblk|awk '/14G/ {print $1}'| head -1)|g" cluster-test.yaml
-    kubectl create -f cluster-test.yaml
+  kubectl create -f https://raw.githubusercontent.com/rook/rook/master/cluster/examples/kubernetes/ceph/common.yaml
+  kubectl create -f https://raw.githubusercontent.com/rook/rook/master/cluster/examples/kubernetes/ceph/crds.yaml
+  kubectl create -f https://raw.githubusercontent.com/rook/rook/master/cluster/examples/kubernetes/ceph/operator.yaml
+  curl https://raw.githubusercontent.com/rook/rook/master/cluster/examples/kubernetes/ceph/cluster-test.yaml -o cluster-test.yaml
+  sed -i "s|#deviceFilter:|deviceFilter: $(lsblk|awk '/14G/ {print $1}'| head -1)|g" cluster-test.yaml
+  kubectl create -f cluster-test.yaml
 }
 
 # wait_for_pod_to_be_ready_state check for operator pod to in ready state
 wait_for_pod_to_be_ready_state() {
-  timeout 20 bash <<-'EOF'
-    until [ $(kubectl get pod -l app=rook-ceph-operator -n rook-ceph -o jsonpath='{.items[*].metadata.name}' -o custom-columns=READY:status.containerStatuses[*].ready | grep -c true) -eq 1 ]; do
+  timeout 200 bash <<-'EOF'
+    until [ $(kubectl get pod -l app=rook-ceph-osd -n rook-ceph -o jsonpath='{.items[*].metadata.name}' -o custom-columns=READY:status.containerStatuses[*].ready | grep -c true) -eq 1 ]; do
       echo "waiting for the pods to be in ready state"
       sleep 1
     done
 EOF
-
 }
+
+########
+# MAIN #
+########
+
+case "$FUNCTION" in
```

Comment:
How about an approach like we have in Rook's [github-action-helper.sh](https://github.com/rook/rook/blob/master/tests/scripts/github-action-helper.sh#L275-L281) to call the function without parsing every function name.